addon/mixins/style-namespacing-extras.js

```diff
@@ -6,6 +6,12 @@ const {
 } = Ember;
 
 export default Mixin.create({
+  _routeIdentifier: computed({
```

Comment:
I don't' think we need this anymore.

Reply:
Oh yeah! That's left over from the previous implementation!